docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -338,7 +243,7 @@ def configure_optimizers(self):
         for seed in range(n_runs):
             pl.seed_everything(seed)
             dataloader_train_ssl, dataloader_train_kNN, dataloader_test = get_data_loaders(batch_size)
-            model = Model(dataloader_train_kNN)
+            model = Model(dataloader_train_kNN, classes)
```

Comment:
Didn't mention it the last time but you could name this `BenchmarkModel` to be a bit clearer.